src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -66,57 +65,44 @@ internal ClientApplicationBase(ApplicationConfiguration config)
         }
 
         async Task<AuthenticationResult> IClientApplicationBaseExecutor.ExecuteAsync(
-            IAcquireTokenSilentParameters silentParameters,
+            AcquireTokenCommonParameters commonParameters,
+            AcquireTokenSilentParameters silentParameters,
             CancellationToken cancellationToken)
```

Comment:
>cancellationToken [](start = 30, length = 17)

as part of this refactoring are we taking the cancellationToken into account for all the flows now?

Reply:
No, I have a work item to plumb it all the way through.  It doesn't have to be done as part of the migration, but the cancellationtoken is exposed in all of our new tokenrequest builder objects so the api is in place.

---
In reply to: [249074577](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/798#discussion_r249074577) [](ancestors = 249074577)